README.md

```diff
@@ -84,30 +84,68 @@ DMeta is an open source Python package that provides ... TODO
 - Run `pip install .`
 
 ## Usage
+### Programmatically 
+#### Clear metadata for a .docx file
+```python
+import os
+from dmeta.functions import clear
+
+DOCX_FILE_PATH = os.path.join(os.getcwd(), "SAMPLE.docx")
+clear(DOCX_FILE_PATH)
+```
+#### Clear metadata for all existing .docx files in the current directory
+```python
+from dmeta.functions import clear_all
+clear_all()
+```
+#### Update metadata for a .docx file
+```python
+import os
+from dmeta.functions import update
+
+CONFIG_FILE_PATH = os.path.join(os.getcwd(), "CONFIG.json") 
+DOCX_FILE_PATH = os.path.join(os.getcwd(), "SAMPLE.docx")
+update(DOCX_FILE_PATH, DOCX_FILE_PATH)
```

Comment:
One of them should be config, I will fix it.

Reply:
shit, yes you are right, the first one I guess.